lib/common/s3.sh

```diff
@@ -68,12 +68,32 @@ s3_put_no_index_keep_file() {
     local src=$1; shift
     local object_name=$1; shift
     local dst=$S3_BUCKET/$object_name
+    local mime=''
 
     test -f $src || file_error "Not a regular file"
 
+    if [[ $src =~ \.gz$ ]]; then
+        mime='application/gzip'
+    elif [[ $src =~ \.nc$ ]]; then
+        mime='application/octet-stream'
+    elif [[ $src =~ \.csv$ ]]; then
+        mime='text/csv'
+    elif [[ $src =~ \.pdf$ ]]; then
+        mime='application/pdf'
+    elif [[ $src =~ \.png$ ]]; then
+        mime='image/png'
+    elif [[ $src =~ \.jpg$ ]]; then
+        mime='image/jpeg'
+    fi
```

Comment:
> Issue is closed now, so it's your problem.

thanks but this is not really constructive. I think no one from the PO team merged this because we just don't know anything re Apache. We have no knowledge on this regard, can't test it and rely on the dev team to tell us if this is good to go or not. So it's up to another Dev to merge this in my opinion.